platform-tests/upstream/run_acceptance_tests.sh

```diff
@@ -7,6 +7,23 @@ if  [ "${DISABLE_CF_ACCEPTANCE_TESTS:-}" = "true" ]; then
   exit 0
 fi
 
+# FIXME: Remove this once we are deploying a version of cf-release
+# that includes capi-release >= 1.35.0.
+(
+  cd  "$(pwd)/cf-release/src/github.com/cloudfoundry/cf-acceptance-tests/"
```

Comment:
I based this off Hector's prior art: https://github.com/alphagov/paas-cf/commit/716384899c2486af26a1a5a463425adfa6e324da

The advantage is all the code in one place I suppose.